lib/omniauth/strategies/hydra0.rb

```diff
@@ -36,6 +36,7 @@ def callback_url
         'email'    => email,
         'nickname' => nickname,
         'name'     => fullname,
+        'roles'    => roles,
```

Comment:
While you're here, can you add *all* the fields that come back?  I think we're missing (looking at `app/routes/consent.js` in profile):

```
      country
      country_code
       postcode
       picture
       profile
       email_verified
```

country/country_code would be useful for showcase as I think we're wanting to be able to filter things by that field (at least) when moderating, so it would be useful to capture and record to the database locally there, rather than calling back to profile each time.

Reply:
so the suggested way of doing it is to set `extra` to everything in `raw_info` - which may suffice and is semi-standard

Reply:
as per https://github.com/omniauth/omniauth-oauth2 that is